polynote-kernel/src/test/scala/polynote/kernel/lang/python/PythonInterpreterSpec.scala

```diff
@@ -11,8 +11,7 @@ class PythonInterpreterSpec extends FlatSpec with Matchers with KernelSpec {
 
   "The python kernel" should "be able to display html using the kernel runtime reference" in {
     assertPythonOutput("kernel.display.html('hi')") { case (vars, output, displayed) =>
-      vars.toSeq should contain only "kernel" -> polynote.runtime.Runtime
-
+      vars.toSeq shouldBe empty
```

Comment:
So, for Python, you don't actually get `kernel` anymore in these results, because it's declared (from Python's POV) in the `init` and not the predef. So it's visible to python, but Python's `kernel` is not visible externally (which is a good thing)

Reply:
Nice!